src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "API Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An API Controller with or without read/write actions",
```

Comment:
We can drop the "an" `API Controller with or without read/write actions`

Reply:
sgty